### DISCUSSION TITLE
Difficulty Launching

### DISCUSSION THREAD

**Discussion by GooseThatCould** [dmdorman/hero6e-foundryvtt#278] [Q&A] [answered]
Hi, is anyone else having difficulty with launching any of the updates post .14? I've tried .15-current and none of the manage to launch properly, leading to a grey screen, however .14 works every time. I was wondering if anyone else has this issue, and might know what it is? I don't want to put it in issues because I'm not sure if its an issue with this system or something else.

**Comment by aeauseth** [comment]
Your the first to bring this to my attention.  Open the browser console (F12 on windows), launch the world and look for any errors.  You can also zip your world up and upload it here for me to analyze.  It may be related to an add-on module, which you can disable prior to world launch by right clicking and choosing launch in safe configuration.

**Reply by GooseThatCould** [reply]
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/123772196/c5d2ab74-a6ee-4f06-b329-a2bb6066a896)
I have no errors, but these warnings. I don't know what Webgl lost means. If none of this matters, I'll post a zip of the game world. Thanks for taking a look.

**Comment by aeauseth** [comment]
Those warnings aren't related to hero system, but do suggest you are using addon modules (which you can disable to see if the problem is related to one of those).  Zip up the world and post it here and I'll look closer.   I may ask for the add-on modules your using.

**Comment by GooseThatCould** [comment]
[ForgeVTT-export-regent-hall-2023-08-16_23_6_20.zip](https://github.com/dmdorman/hero6e-foundryvtt/files/12364123/ForgeVTT-export-regent-hall-2023-08-16_23_6_20.zip)
I am using 36 modules, they seem to work with .14 but not afterwards. I guess one of them could be conflicting with whatever was updated/changed.
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/123772196/74fcbd14-f8cb-4ad4-be27-3644a891b299)
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/123772196/5d21de2a-e420-4002-bfca-169cbc7e1443)

**Comment by aeauseth** [comment]
I'm able to load your world and all the scenes without issue, but I have no modules installed/active.  So the problem is likely one of the modules or a compatibility issue with one of the modules and hero system.  You have lots of modules.  Deactivea All Modules, and see if the world loads.  If it does, the re-activate 1 at a time, reloading between each one until you find the problem.  Let me know which module is the problem and I'll look at compatibility issues with that module.  I recommend starting with the modules with yellow exclamation marks as those are unvetted with the current stuff.

**Comment by GooseThatCould** [comment]
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/123772196/c8ba53f3-6287-4ee8-847c-2b9bdd0fecdc)
I've tried going through it all, and eventually I got to the point where this was the only module installed, and unfortunately I'm still getting the same results, .14 loads fine but not .22. I've tried launching in safe configuration too just to try, with the same results as well. I'm not sure now what could be the problem.

**Comment by GooseThatCould** [comment]
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/123772196/5f9e4259-16b1-4fab-906f-60e5e6a357d3)
This is what I get when I press F12

**Comment by aeauseth** [comment]
Try closing down Foundry and launch it again with your world, with only THE FORGE active.

**Comment by aeauseth** [comment]
You can use https://get.webgl.org/webgl2/ to make sure your browser is configured to use WebGL2.  Not technically a requirement, but you will see poor performance if WebGL isn't working.  A computer reboot might also be a good diagnostic test.

**Comment by GooseThatCould** [comment]
I've tried both of these things, (using firefox and checked and I see the spinning cube) and unfortunately... it still won't load .22
There are no other tabs open, no other nothing, restarted my computer and went straight to try it. I'm kinda lost.

**Comment by GooseThatCould** [comment]
Loads to 100% and grey screens

**Comment by GooseThatCould** [comment]
Tried creating a whole new world, unfortunately this one doesn't seem to work either.

**Comment by aeauseth** [comment]
Trying a new world on my system.  Which hero system do you currently have installed?

**Comment by aeauseth** [comment]
If your on discord you can PM me **aaron3736**, then we can voice chat and you can share your screen.  Perhaps we can figure it out that way.

**Reply by GooseThatCould** [reply]
I've sent the request, goosethatcould

**Comment by aeauseth** [comment, marked as answer]
Turned out to be a Firefox issue.  Works with Chrome.

**Comment by aeauseth** [comment]
Installed Firefox and I was able to reproduce the issue.  Gets stuck at 98% when loading new scene.  No console errors.
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/13549766/3ba357c6-3996-44c5-b922-7bcff3f4e627)

**Comment by aeauseth** [comment]
Firefox has a known issue.  Solution is to add height & width properties to all custom SGV files.  This should be resolved in version 3.0.24, however the core underlying Firefox issue is still present and I'll need to remember to add height/width to all future SGV files.

REF: https://github.com/foundryvtt/foundryvtt/issues/9052#issuecomment-1465237676

```Uncaught Error: The SVG image must have width and height defined (in pixels), canvas API needs them.
    onload index.js:126
    _loadSvg index.js:126
    _load index.js:126
    load index.js:126
    Bs index.js:126
    parse earcut.js:184
    promise earcut.js:128
    _getLoadPromiseAndParser earcut.js:128
    h earcut.js:128
    load earcut.js:128
    _mapLoadToResolve earcut.js:184
    load earcut.js:184
    loadAsset foundry.js:29250
    loadTexture foundry.js:29256
    loadTexture foundry.js:29220
    load foundry.js:29231
    loadSceneTextures foundry.js:29196
    draw foundry.js:26510
    view foundry.js:23615
    initializeCanvas foundry.js:8815
    setupGame foundry.js:8644
    _initializeGameView foundry.js:9908
    _initializeView foundry.js:9884
    initialize foundry.js:8570
    async* foundry.js:91546
    async* foundry.js:91522
index.js:126
```

**Comment by aeauseth** [comment]
![image](https://github.com/dmdorman/hero6e-foundryvtt/assets/13549766/e0bf3590-5e3e-4596-b41f-d92b6ced79fc)
